router/pkg/plan_generator/testdata/plans/base/4.graphql

```diff
@@ -0,0 +1,26 @@
+QueryPlan {
+  Subscription {
+    Primary: {
+      Fetch(service: "2-nats-3") {
```

Comment:
It might be a bit tricky, but doable, to know what fields were selected from the EDFS source. We could split this into another story and add the root selections that belong to the EDFS source, just to make it easier to understand what's going on.

Reply:
Created a separate story for this.